### DISCUSSION TITLE
Island Mission

### DISCUSSION THREAD

**Discussion by nivcoo** [BG-Software-LLC/SuperiorSkyblock2#833] [Pending Feature Requests]
Hello, can you add an option in mission file to add number in total of the quest requirement for number of players members, exemple : 

Break 100 potatoes if your only member but if you're two, you have to break 120 potatoes because you have set 20 to "extra count" option. I don't know iff is it clearly.

**Comment by nivcoo** [comment]
Any update or comment ?

**Reply by OmerBenGera** [reply]
> Any update or comment ?

There's nothing to comment.
If someone makes a PR, it will be accepted, or if I have time...
For now, I don't have time and therefore I don't implement it myself :)
